cli/Valet/Site.php

```diff
@@ -570,7 +570,7 @@ public function createCa(int $caExpireInDays): void
         ));
 
         $this->cli->runAsUser(sprintf(
-            'openssl req -new -newkey rsa:2048 -days %s -nodes -x509 -subj "/C=/ST=/O=%s/localityName=/commonName=%s/organizationalUnitName=Developers/emailAddress=%s/" -keyout "%s" -out "%s"',
+            'openssl req -new -newkey rsa:2048 -days %s -nodes -x509 -subj "/C=/ST=/O=%s/localityName=/commonName=%s/organizationalUnitName=Developers/emailAddress=%s/" -keyout "%s" -out "%s" -addext "basicConstraints=critical,CA:TRUE" -addext "keyUsage=critical,digitalSignature,keyCertSign" -addext "subjectKeyIdentifier=hash"',
```

Comment:
I'm not an expert in this subject matter, but while looking at this change I notice here you added `basicConstraints=critical,CA:TRUE`, but in the `openssl.conf` file it's `basicConstraints = critical,CA:FALSE`. The TRUE vs FALSE stood out. 
Was that intentional? Is it moot? Just curious. And commenting "just in case" it matters.